src/main/resources/com/openshift/jenkins/plugins/OpenShiftDSL.groovy

```diff
@@ -384,8 +384,8 @@ class OpenShiftDSL implements Serializable {
                 project:(getProject ? currentContext.getProject() : null),
                 verb:verb,
                 verbArgs:verbArgs,
-                userArgs:userArgsList,
                 options:optionsBase,
+                userArgs:userArgsList,
```

Comment:
@coreydaley - are you sure this reconciles with the `ClientCommandBuilder` constructor?

There, the order is verbArgs, options, userArgs ...

It might better to pass the skip tls verification as a separate param just like server, token, project, and then adjust ClientCommandBuilder.buildCommand to prepend like we did with those

I'm also curious about the server certificate auth option --certificat-authority ... it is added to verboseOptionsBase, but then in ClientCommandBuilder.buildCommand ... that command is only added if !redacted or loglevel is greater than 0 ... I would think we would want to add that regardless, but perhaps not redact based on redacted/loglevel.  Please look into that as well.